### PR TITLE
Clean first and last name - case and trimming

### PR DESCRIPTION
**Pull request by karol-gro**

### CODE REVIEW

File: automation/Zhp.Office.AccountManagement/Zhp.Office.AccountManagement.Tests/Domain/Model/ActivationRequestTests.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    [Theory]
+    [InlineData("Jan", "Jan")]
+    [InlineData("ANNA", "Anna")]
+    [InlineData("renée", "Renée")]
+    [InlineData("ToMaSz", "Tomasz")]
+    [InlineData("bARTŁOMIEJ", "Bartłomiej")]
+    [InlineData("JAN KANTY", "Jan Kanty")]
+    [InlineData("øyvind", "Øyvind")]
+    [InlineData("JEAN-BAPTISTE", "Jean-Baptiste")]
+    [InlineData("von vordon", "Von Vordon")] // acceptable, but wrong
```
Comment (karol-gro) [inline automation/Zhp.Office.AccountManagement/Zhp.Office.AccountManagement.Tests/Domain/Model/ActivationRequestTests.cs:18]: tu jest taka trudność, że pewne części nazwisk jak _von_ czy _de_ powinny być z małych liter, ale to jest taki edge case, że bym olał

**Comment by karol-gro** [comment]
Jeszcze w sumie muszę ten cleanup wywołać, ale to jak wrócę do kompa

**Review by norberto5 — APPROVED** [review]
